packages/nuxt/src/module.ts

```diff
@@ -0,0 +1,25 @@
+import { addComponent, defineNuxtModule } from "@nuxt/kit";
+import * as onyx from "sit-onyx";
```

Comment:
Do you know if this is performant and stree-shakeable for bundle size? Or would we need a dynamic import for this?

Reply:
Everything you see in this file will only run at build time, that's how nuxt modules work. It only registers the modules so they can be auto imported by nuxt. The import here is only used to get the names of the components for that. If one of the components is used in e.g. a page Nuxt will add `import { OnyxComponentName } from "sit-onyx"` to the page at build time. So all of this has nothing to do with the final bundle or the runtime.